x/programs/cmd/simulator/cmd/root.go

```diff
@@ -114,23 +116,24 @@ func (s *simulator) Init() error {
 	s.log, err = logFactory.Make("simulator")
 	if err != nil {
 		logFactory.Close()
-		return nil
+		return err
 	}
 
 	sk, err := bls.NewSecretKey()
 	if err != nil {
-		return nil
+		return err
 	}
 
 	// setup pebble and db manager
-	pdb, _, err := pebble.New(dbPath, pebble.NewDefaultConfig())
+	// pdb, _, err := pebble.New(dbPath, pebble.NewDefaultConfig())
+	pdb := memdb.New()
```

Comment:
just curious, should we update the comment and remove the pebble instantiation?

Reply:
Yes, will do!

Reply:
Might actually be able to just use `nil`, not sure if this is actually used, will test.